histomicstk/PodoSighter_cnn_folder/Complete_CNN_Prediction.py

```diff
@@ -204,10 +237,19 @@
 f = open(args.outxml1, 'wb')
 f.write(xml_data)
 f.close()
+gc.uploadFileToItem(itemID, args.outxml1, reference=None, mimeType=None, filename=None, progressCallback=None)
 
 tree = ET.parse(args.outxml1)
 root = tree.getroot()
 
 json_data = xmltojson(root)
 with open(args.jsonout, 'w') as annotation_file:
     json.dump(json_data, annotation_file, indent=2, sort_keys=False)
+gc.uploadFileToItem(itemID, args.jsonout, reference=None, mimeType=None, filename=None, progressCallback=None)

```

Comment:
Ditto previous comment? Why we're uploading here?

Reply:
I wanted to get your confirmation before I remove this part. Removed in latest commit